src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/EventTests.cs

```diff
@@ -161,8 +161,8 @@ public async Task LogEventWithCorrelationInfo_SinksToApplicationInsights_Results
                         Assert.Equal(operationId, actualOperationId);
                         Assert.Equal(transactionId, actualTransactionId);
                         Assert.Equal(operationParentId, actualOperationParentId);
-                        Assert.Equal(operationId, trace.Operation.Id);
-                        Assert.Equal(operationParentId, trace.Operation.ParentId);
+                        Assert.Equal(transactionId, trace.Operation.Id);
+                        Assert.Equal(operationId, trace.Operation.ParentId);
```

Comment:
I might be missing some context, but I think this is strange ?

Reply:
This is the conclusion from the service-to-service PR we had last week. The non-requests will need some different telemetry correlation to be fully linked to the incoming requests.

Also, Application Insights' `Operation ID` = `Transaction ID` from Arcus.

Reply:
😄 I can only imagine what that would be like, constantly switching. You're doing a fine job! 👍